app/front/scripts/controllers/main.js

```diff
@@ -244,23 +305,13 @@
               filter: cut
             };
 
-            $scope.state.babbageTreeMap = {
```

Comment:
Because old babbage.ui required different parameters for each vis. After refactoring, Igor unified iterface for components, so this code now is never used